pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -176,6 +176,20 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Update dynamic log4j2 logger level in runtime by classname.")
+    private class UpdateLoggerLevelCmd extends CliCommand {
```

Comment:
Please note that there is no control about the broker who is serving the request.

Can we report this to the user?
- report the id of the broker that process the request
- write a generic warning

Reply:
🤔 Maybe not.
in product environment, `authenticationEnabled = true` as usual ,so only developers or admin can do this. 
in addition, user call this request will mark the broker ip in log as follow:
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/16517186/154221713-efb4d2d2-5a79-4d47-adf6-514fdf70c957.png">